packages/ui/src/components/tabs.tsx

```diff
@@ -130,12 +130,7 @@ const TabsTrigger = React.forwardRef<React.ElementRef<typeof TabsPrimitive.Trigg
         {...props}
       >
         {context.variant === 'navigation' && (
-          <span
-            className="absolute -inset-x-[30px] -inset-y-5 hidden group-data-[state=active]:block"
-            style={{
-              background: 'radial-gradient(50% 50% at 50% 50%, rgba(48, 48, 54, 0.3) 0%, rgba(48, 48, 54, 0) 100%)'
-            }}
-          />
+          <span className="bg-tab-gradient-radial absolute -inset-x-[30px] -inset-y-5 hidden group-data-[state=active]:block" />
```

Comment:
Adding `bg-tab-gradient-radial` class is the only change in this file. Other updates was done by prettier.